src/tkn.ts

```diff
@@ -958,22 +960,39 @@ export class TknImpl implements Tkn {
     );
     if (result.stdout.trim() === 'no') {
       const tknDownMsg = 'The current user doesn\'t have the privileges to interact with tekton resources.';
+      disableWatch();
       return [new TektonNodeImpl(null, tknDownMsg, ContextType.TKN_DOWN, this, TreeItemCollapsibleState.None)];
     }
     if (result.error && getStderrString(result.error).indexOf('You must be logged in to the server (Unauthorized)') > -1) {
       const tknMessage = 'Please login to the server.';
+      disableWatch();
       return [new TektonNodeImpl(null, tknMessage, ContextType.TKN_DOWN, this, TreeItemCollapsibleState.None)]
     }
     if (result.error && getStderrString(result.error).indexOf('the server doesn\'t have a resource type \'pipeline\'') > -1) {
       const tknDownMsg = 'Please install the OpenShift Pipelines Operator.';
+      disableWatch();
       return [new TektonNodeImpl(null, tknDownMsg, ContextType.TKN_DOWN, this, TreeItemCollapsibleState.None)];
     }
     const serverCheck = RegExp('Unable to connect to the server');
     if (serverCheck.test(getStderrString(result.error))) {
       const loginError = 'Unable to connect to OpenShift cluster, is it down?';
+      disableWatch();
       return [new TektonNodeImpl(null, loginError, ContextType.TKN_DOWN, this, TreeItemCollapsibleState.None)];
     }
 
+    if ((await version())?.trigger === undefined || (await version()).trigger.trim() === 'unknown') {
```

Comment:
You execute async function `version`(which doing requests to k8s cluster!) twice just to check it result, it will be much better if you put result of it in local const and do all validation on it.